scripts/monorepo/src/getDependencies.spec.js

```diff
@@ -92,6 +87,11 @@ describe(`#getDependencies`, () => {
           "isTopLevel": false,
           "name": "scripts-prettier",
         },
+        Object {
```

Comment:
correctly returns also scripts-cypress which was previously missing in one of react-text devDeps